packages/util/src/BigIntMath.ts

```diff
@@ -1,7 +1,12 @@
+import { BigNumber } from 'bignumber.js';
+
 export const BigIntMath = {
   abs(x: bigint): bigint {
     return x < 0n ? -x : x;
   },
+  divide(numerator: bigint, denominator: bigint): string {
```

Comment:
I'm actually not sure if it belongs here. `bigint` is supposed to be an integer (as the name implies) and integer math division is a little different. @rhyslbw do you still think it's a good idea to have it here? I'm leaning towards rolling rolling this back to put usage of `bignumber.js` where it's needed directly. 

Otherwise, we should come up with a better name for this function. Maybe `floatDivision`?

Reply:
Yeah, my request wasn't considering that this module was originally intended to be based on `Math`, so probably just revert and move the `bignumber.js` dependency back to `cardano-services`